src/ViewHelpers/IfViewHelper.php

```diff
@@ -12,36 +12,50 @@
 
 /**
  * This ViewHelper implements an if/else condition.
+ * The code entered within this ViewHelper is then only executed
+ * or shown, when the condition matches.
  *
- * Conditions:
+ * Fluid Boolean Rules / Conditions:
+ * =================================
+ *
+ * A condition is evaluated as a boolean value, so you can use any boolean argument,
+ * like a variable.
+ * Alternatively, you can use a full boolean expression.
+ * The entered expression is evaluated as a PHP expression. You can combine multiple
+ * expressions via :php:`&&` (logical AND) and :php:`||` (logical OR).
+ *
+ * Have a look into the Fluid section of the "TYPO3 Explained" Documentation for
+ * more details about complex conditions.
  *
- * As a condition is a boolean value, you can just use a boolean argument.
- * Alternatively, you can write a boolean expression there.
- * The fluid expression is evaluated as PHP expression without any change.
  * Boolean expressions have the following form:
  *
- * !{variable}
+ * (!){variable}
  *
  * ::
  *
+ *       <f:if condition="{foo}">
+ *           Will be shown if foo is truthy.
+ *       </f:if>
+ *
  *       <f:if condition="!{foo}">
  *           Will be shown if foo is falsy.
  *       </f:if>
  *
  * or
  *
- * XX Comparator YY
+ * ::
+ *       XX Comparator YY
  *
- * Comparator is one of: ==, !=, <, <=, >, >= and %
- * The % operator converts the result of the % operation to boolean.
+ * Comparator is one of: :php:`==, !=, <, <=, >, >=` and :php:`%`
+ * The :php:`%` operator (bitmask) converts the result of the operation to boolean.
```

Comment:
`%` is the modulo operator. Not sure if this is related to bitmask, but it should be called modulo, I think.